project/Template-Folder/index.rst

```diff
@@ -1,4 +1,4 @@
-.. This is a template top-level index file for a directory in the procedure's arm of the documentation
+.. This is a template top-level index file for a directory in the procedure's arm of the documentation. Each procedure will have its own sub-directory. This comment will be deleted when the template folder is copied to the destination.
```

Comment:
This comment *may* be deleted.....

Reply:
OK - I'll make that change across the board.